examples/broadcast/next.config.js

```diff
@@ -8,15 +8,15 @@ module.exports = {
     return config;
   },
   // TODO: pass dialect env as env variable and enable proxy only on local-development
-  // rewrites:
-  //   process.env.NODE_ENV == 'development'
-  //     ? async () => {
-  //         return [
-  //           {
-  //             source: '/:path*',
-  //             destination: 'http://localhost:8080/:path*',
-  //           },
-  //         ];
-  //       }
-  //     : undefined,
+  rewrites:
+    process.env.NODE_ENV == 'development'
+      ? async () => {
+          return [
+            {
+              source: '/:path*',
+              destination: 'http://localhost:8080/:path*',
```

Comment:
what's the use case for this?

Reply:
hm, I thought that data-service has cors disabled!
no action required, just was curious why it's like this ;)